src/transformers/configuration_utils.py

```diff
@@ -144,6 +144,10 @@ class PretrainedConfig(PushToHubMixin):
         top_p (`float`, *optional*, defaults to 1):
             Value that will be used by default in the `generate` method of the model for `top_p`. If set to float < 1,
             only the most probable tokens with probabilities that add up to `top_p` or higher are kept for generation.
+        typical_p (`float`, *optional*, defaults to 1):
+            Locally typical sampling orders tokens according to how similar the negative log probability of predicting a
+            specific token is to the expected conditional entropy of a random next token. The smallest set of most
```

Comment:
I'd keep this similar to the other `typical_p` docstring.

```suggestion
            specific token is to the expected conditional entropy of a random next token. If set to float < 1, the smallest set of most
```

Reply:
Okay! I replaced the docstring in the PretrainedConfig with the one from GenerationConfig!